lib/src/cnf/mod.rs

```diff
@@ -35,3 +35,9 @@ pub const SERVER_NAME: &str = "SurrealDB";
 
 /// Datastore processor batch size for scan operations
 pub const PROCESSOR_BATCH_SIZE: u32 = 50;
+
+/// Set the maximum WebSocket frame size to 16mb
```

Comment:
Is this comment relevant? 🤔

Reply:
oh whoops, I absolutely did not copy that from elsewhere 😛